pages/data_life_cycle/processing.md

```diff
@@ -1,52 +1,46 @@
-# training:
-#   - name: Training in TeSS
-#     registry: TeSS
-#     url: https://tess.elixir-europe.org/search?q=%22data+process%22#materials
-# dsw:
-#   - name:
-#       List the data formats you will be using for interpretation and describe their
-#       structure
-#     uuid: a797cab9-0829-4787-a096-1b5cedc9147f
-#   - name: How will you work with your data?
-#     uuid: df36fb68-131c-4f31-a42b-684abf523bbc
 ---
 
-## What is data processing?
+## What is data storage?
 
-Data processing is the phase in the project where data is converted into a desired format and prepared for analysis. When data has been freshly collected, data processing includes some automated steps in a workflow that perform format conversion, quality check and preprocessing following a standardised protocol. The main aim of processing is to:
+Data storage is the phase in the project where data is converted moved and saved into a safe storage solution. Storing your data in a safe location, helps you manage different risks, like data breaches, loosing your data, and data corruption. 
 
-- convert data into readable format giving it the shape and form necessary for downstream analysis;
-- discard bad or low quality data in order to create clean, high-quality dataset for reliable results.
+Most of the safe storage solutions have back-up systems that use a synked version of your files and store them automatically in a different location, this is in case one server fails, you can accees your data in the back-up location. 
 
-When data is imported from existing sources, e.g. data to be reused from another project, processing can also include manual steps to make it suitable for analysis. These steps include but are not limited to:
+<!-- ERIM focused information (begin) -->
+## What storage solutions do I have as an ERIM member? 
 
-- making changes to data formats such that different datasets will be compatible for integration with each other;
-- changing coding systems or ontologies for the data to bring everything to the same level;
-- filtering data such that only data suitable for the project is retained.
+At EUR we have several storage solutions, depending on the type of data you are pocessing.
 
-After data processing, clean data is ready for analysis and should therefore be available to the members of the project team that need to perform the next steps.
+| **Storage Solution** | **Data type** | **Access Management** | 
+| ----- | --- | --- |
+| EUR Dropbox | Public data | Can be shared with externals|
+| EUR Yoda | Personal (senstive) data | Can be shared with externals|
+| EUR Document Vault | Personal  (highly senstive)  data | Can be shared with externals|
+| EUR SURF Research Drive | Personal (senstive) data | Can be shared with externals|
+| EUR OneDrive | Personal (senstive) data | Can only be shared within EUR|
+| EUR SURFdrive | Personal data | Can only be shared within Dutch universities|
```

Comment:
I got inspired by this table, and here's the vision that I built on. Specifically:

1. I reordered things, so that a more recommended solution comes to the top, and the least recommended comes to the bottom (e.g., Dropbox)
2. I tried using the icons to simplify the information presented, and renamed the columns "Data Type" and "Access Management" to "Personal/Sensitive Data" and "Collaborating Outside EUR"
3. I added EUR SharePoint/Teams as one of the options

I hope I captured the original information—curious to hear what you think! 

![image](https://github.com/eur-nl/erim-research-toolbox/assets/17035406/87b5cca3-19ae-4c49-89e8-920fba47c8cd)


```suggestion
| **Storage Solution**    | **Personal/Sensitive Data** | **Collaborating Outside EUR**    |
| ----------------------- | --------------------------- | -------------------------------- |
| EUR Yoda                | ✅                          | ✅                               |
| EUR Document Vault      | ✅ most secure              | ✅                               |
| EUR SURF Research Drive | ✅                          | ✅                               |
| EUR SharePoint/Teams          | ✅                          | ☑️ with guest access             |
| EUR OneDrive            | ✅                          | ❌                               |
| EUR SURFdrive           | ☑️ only personal data       | ☑️ only among Dutch universities |
| EUR Dropbox             | ❌                          | ✅                               |
```

Reply:
I really like this! :)